CHANGES.rst

```diff
@@ -22,6 +22,11 @@ Olivier Grisel
     thread is not the Python default. Thanks to Roman Yurchak for the
     suggestion.
 
+Karan Desai
+
+    py.test is used to run the tests instead of nosetests.
+    ``python setup.py test`` or ``python setup.py nosetests`` do not work
```

Comment:
I'll take a closer look in a few hours if I can, otherwise tomorrow. It does seem very close to being merged.